  INFO root [0x00003138] [2018-05-01 12:12:31,827] - 
  INFO root [0x00003138] [2018-05-01 12:12:31,885] - -----------------------------------------------------------------------------------------


  INFO root [0x00003138] [2018-05-01 12:12:33,222] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00003138] [2018-05-01 12:12:33,223] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00003138] [2018-05-01 12:12:33,224] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00003138] [2018-05-01 12:12:33,724] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00003138] [2018-05-01 12:12:33,770] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00003138] [2018-05-01 12:12:33,774] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:50736 )
  INFO root [0x00003138] [2018-05-01 12:12:33,774] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:50736, SOMAXCONN=128.
  INFO root [0x00003138] [2018-05-01 12:12:33,774] - server-configs:
  INFO root [0x00003138] [2018-05-01 12:12:33,775] - 	gameUpdateHertz : 10
  INFO root [0x00003138] [2018-05-01 12:12:33,775] - 	entryScriptFile : kbemain
  INFO root [0x00003138] [2018-05-01 12:12:33,775] - 	internalAddr : 192.168.0.106:50736
  INFO root [0x00003138] [2018-05-01 12:12:33,775] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00003138] [2018-05-01 12:12:33,776] - 	componentID : 7001
  INFO root [0x00003138] [2018-05-01 12:12:35,374] - Components::findLogger: found logger, addr:192.168.0.106:50721
  INFO root [0x00003138] [2018-05-01 12:12:35,375] - Components::addComponent: reset orderLog, uid=17503!
  INFO root [0x00003138] [2018-05-01 12:12:35,375] - Components::addComponent[logger], uid=17503, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00003138] [2018-05-01 12:12:35,704] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 17503. PID: 12596 ----
  INFO root [0x00003138] [2018-05-01 12:12:35,705] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00003138] [2018-05-01 12:12:35,739] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00003138] [2018-05-01 12:12:35,792] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00003138] [2018-05-01 12:12:40,542] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003138] [2018-05-01 12:12:40,584] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00003138] [2018-05-01 12:12:40,648] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00003138] [2018-05-01 12:12:41,292] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003138] [2018-05-01 12:12:41,373] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00003138] [2018-05-01 12:12:41,422] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003138] [2018-05-01 12:12:41,471] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003138] [2018-05-01 12:12:41,743] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00003138] [2018-05-01 12:12:42,616] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00003138] [2018-05-01 12:12:42,616] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00003138] [2018-05-01 12:12:43,713] - loaded entity-script:Account(1).
  INFO root [0x00003138] [2018-05-01 12:12:43,714] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00003138] [2018-05-01 12:12:43,773] - TelnetServer server is running on port 40000
  INFO root [0x00003138] [2018-05-01 12:12:44,219] - ---- baseapp is running ----
 DEBUG root [0x00003138] [2018-05-01 12:12:44,250] - Components::process(): Request for the process of identity...
  WARN root [0x00003138] [2018-05-01 12:12:50,408] - Forwarding logs to logger[192.168.0.106:50721/0/0/0]...
 ERROR root [0x00003138] [2018-05-01 12:57:57,018] - Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  INFO root [0x00003138] [2018-05-01 12:57:57,029] - The following logs sent to logger failed:
  WARN root [0x00003138] [2018-05-01 12:57:57,029] - ==> [2018-05-01 12:57:57 017] TCPPacketReceiver::processPendingEvents(192.168.0.106:50721): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003138] [2018-05-01 12:57:57,029] - ==> [2018-05-01 12:57:57 019] Components::removeComponentByChannel: logger : 2000, Abnormal exit! errorno=10054
  WARN root [0x00003138] [2018-05-01 12:57:57,076] - TCPPacketReceiver::processPendingEvents(192.168.0.106:50723): Throwing REASON_GENERAL_NETWORK - WSAECONNRESET
 ERROR root [0x00003138] [2018-05-01 12:57:57,078] - Components::removeComponentByChannel: dbmgr : 4000, Abnormal exit! errorno=10054
  INFO root [0x00003138] [2018-05-01 12:57:57,103] - Shutdowner::onShutdownBegin: shutting down(period=0, tickPeriod=1)
S_INFO root [0x00003138] [2018-05-01 12:57:57,103] - onBaseAppShutDown: state=0
  INFO root [0x00003138] [2018-05-01 12:57:57,103] - Shutdowner::onShutdown: shutting down
S_INFO root [0x00003138] [2018-05-01 12:57:57,103] - onBaseAppShutDown: state=1
  INFO root [0x00003138] [2018-05-01 12:57:57,103] - Shutdowner::onShutdownEnd: shutting down
S_INFO root [0x00003138] [2018-05-01 12:57:57,104] - onBaseAppShutDown: state=2
 DEBUG root [0x00003138] [2018-05-01 12:57:57,209] - Script::uninstall_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00003138] [2018-05-01 12:57:57,209] - Script::uninstall_py_dlls(): PyInit__socket
 DEBUG root [0x00003138] [2018-05-01 12:57:57,210] - Script::uninstall_py_dlls(): PyInit__ssl
 DEBUG root [0x00003138] [2018-05-01 12:57:57,210] - Script::uninstall_py_dlls(): PyInit__hashlib
 DEBUG root [0x00003138] [2018-05-01 12:57:57,210] - Script::uninstall_py_dlls(): PyInit_select
 DEBUG root [0x00003138] [2018-05-01 12:57:57,210] - Script::uninstall_py_dlls(): PyInit__ctypes
 DEBUG root [0x00003138] [2018-05-01 12:57:57,210] - Script::uninstall_py_dlls(): PyInit__elementtree
 DEBUG root [0x00003138] [2018-05-01 12:57:57,210] - Script::uninstall_py_dlls(): PyInit_unicodedata
  INFO root [0x00003138] [2018-05-01 12:57:57,248] - Script::uninstall(): is successfully!
 DEBUG root [0x00003138] [2018-05-01 12:57:57,248] - ThreadPool::destroy(): starting size 2.
  INFO root [0x00001d30] [2018-05-01 12:57:58,289] - 
  INFO root [0x00001d30] [2018-05-01 12:57:58,292] - -----------------------------------------------------------------------------------------


  INFO root [0x00001d30] [2018-05-01 12:57:58,292] - Resmgr::initialize: KBE_ROOT=E:/kbengine/
  INFO root [0x00001d30] [2018-05-01 12:57:58,293] - Resmgr::initialize: KBE_RES_PATH=E:/kbengine/kbe/res/;E:/kbengine/assets/;E:/kbengine/assets/scripts/;E:/kbengine/assets/res/
  INFO root [0x00001d30] [2018-05-01 12:57:58,293] - Resmgr::initialize: KBE_BIN_PATH=E:/kbengine/kbe/bin/server/
  INFO root [0x00001d30] [2018-05-01 12:57:58,301] - NetworkInterface::initialize(EXTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:20015 )
  INFO root [0x00001d30] [2018-05-01 12:57:58,301] - NetworkInterface::initialize(EXTERNAL): address 192.168.0.106:20015, SOMAXCONN=128.
  INFO root [0x00001d30] [2018-05-01 12:57:58,302] - NetworkInterface::initialize(INTERNAL): bound to all interfaces with default route interface on 192.168.0.106 ( 192.168.0.106:51110 )
  INFO root [0x00001d30] [2018-05-01 12:57:58,302] - NetworkInterface::initialize(INTERNAL): address 192.168.0.106:51110, SOMAXCONN=128.
  INFO root [0x00001d30] [2018-05-01 12:57:58,302] - server-configs:
  INFO root [0x00001d30] [2018-05-01 12:57:58,303] - 	gameUpdateHertz : 10
  INFO root [0x00001d30] [2018-05-01 12:57:58,303] - 	entryScriptFile : kbemain
  INFO root [0x00001d30] [2018-05-01 12:57:58,303] - 	internalAddr : 192.168.0.106:51110
  INFO root [0x00001d30] [2018-05-01 12:57:58,303] - 	externalAddr : 192.168.0.106:20015
  INFO root [0x00001d30] [2018-05-01 12:57:58,303] - 	componentID : 7001
  WARN root [0x00001d30] [2018-05-01 12:57:58,832] - BundleBroadcast::BundleBroadcast: Cannot bind to port 20975, errorno=10048
  INFO root [0x00001d30] [2018-05-01 12:57:58,835] - ---- baseapp Version: 2.0.0. ScriptVersion: 0.1.0. Protocol: F9DFFB319FBADF35943ADF4D42C4718C. Config: Debug X86. Built: 15:53:31 Apr 29 2018. AppID: 7001. UID: 26543. PID: 12800 ----
  INFO root [0x00001d30] [2018-05-01 12:57:58,836] - Server : Intel(R) Xeon(R) CPU E3-1230 v3 @ 3.30GHz with 4194303kB RAM
  INFO root [0x00001d30] [2018-05-01 12:57:58,836] - ThreadPool::createThreadPool: creating  threadpool...
  INFO root [0x00001d30] [2018-05-01 12:57:58,836] - ThreadPool::createThreadPool: successfully(2), newThreadCount=1, normalMaxThreadCount=2, maxThreadCount=8
 DEBUG root [0x00001d30] [2018-05-01 12:57:58,959] - Script::install_py_dlls(): PyInit_pyexpat
 DEBUG root [0x00001d30] [2018-05-01 12:57:58,959] - Script::install_py_dlls(): PyInit__socket
 DEBUG root [0x00001d30] [2018-05-01 12:57:58,960] - Script::install_py_dlls(): PyInit__ssl
 DEBUG root [0x00001d30] [2018-05-01 12:57:58,964] - Script::install_py_dlls(): PyInit__hashlib
 DEBUG root [0x00001d30] [2018-05-01 12:57:58,964] - Script::install_py_dlls(): PyInit_select
 DEBUG root [0x00001d30] [2018-05-01 12:57:58,964] - Script::install_py_dlls(): PyInit__ctypes
 DEBUG root [0x00001d30] [2018-05-01 12:57:58,965] - Script::install_py_dlls(): PyInit__elementtree
 DEBUG root [0x00001d30] [2018-05-01 12:57:59,001] - Script::install_py_dlls(): PyInit_unicodedata
  INFO root [0x00001d30] [2018-05-01 12:57:59,035] - Script::install(): is successfully, Python=(3.4.2 (default, Apr 29 2018, 15:53:05) [MSC v.1913 32 bit (Intel)])!
  INFO root [0x00001d30] [2018-05-01 12:57:59,035] - Script::install(): KBExtra is successfully!
 DEBUG root [0x00001d30] [2018-05-01 12:57:59,041] - loaded entity-script:Account(1).
  INFO root [0x00001d30] [2018-05-01 12:57:59,041] - Baseapp::initializeEnd: started resmgr tick(60s)!
  INFO root [0x00001d30] [2018-05-01 12:57:59,041] - TelnetServer server is running on port 40000
  INFO root [0x00001d30] [2018-05-01 12:57:59,319] - ---- baseapp is running ----
 DEBUG root [0x00001d30] [2018-05-01 12:57:59,319] - Components::process(): Request for the process of identity...
  WARN root [0x00001d30] [2018-05-01 12:58:05,389] - EntityApp::checkTickPeriod: tick took 6070% (6.07 seconds)!
 DEBUG root [0x00001d30] [2018-05-01 12:58:06,120] - Baseapp::lookApp: 192.168.0.106:51127/0/0/0
  INFO root [0x00001d30] [2018-05-01 12:58:06,420] - Components::findComponents: find logger(1)...
  INFO root [0x00001d30] [2018-05-01 12:58:06,477] - Components::findComponents: found logger, addr:192.168.0.106:51109
  INFO root [0x00001d30] [2018-05-01 12:58:06,477] - Components::addComponent: reset orderLog, uid=26543!
  INFO root [0x00001d30] [2018-05-01 12:58:06,478] - Components::addComponent[logger], uid=26543, componentID=2000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001d30] [2018-05-01 12:58:06,480] - Components::findComponents: find dbmgr(2)...
  INFO root [0x00001d30] [2018-05-01 12:58:06,584] - Components::findComponents: found dbmgr, addr:192.168.0.106:51115
  INFO root [0x00001d30] [2018-05-01 12:58:06,584] - Components::addComponent: reset orderLog, uid=26543!
  INFO root [0x00001d30] [2018-05-01 12:58:06,585] - Components::addComponent[dbmgr], uid=26543, componentID=4000, globalorderid=1, grouporderid=1, totalcount=1
  INFO root [0x00001d30] [2018-05-01 12:58:06,585] - Components::findComponents: found dbmgr, addr:192.168.0.106:51115
  WARN root [0x00001d30] [2018-05-01 12:58:07,585] - EntityApp::checkTickPeriod: tick took 1264% (1.26 seconds)!
  WARN root [0x00001d30] [2018-05-01 12:58:07,586] - Forwarding logs to logger[192.168.0.106:51109/0/0/0]...
